### ISSUE TITLE
Exception showing PR

### ISSUE THREAD

**Issue by hanke** [sociomantic-tsunami/git-hub#175]
For this one:

https://github.com/datalad/datalad/pull/388

I get that one:

````
% git-hub pull show 388

#388: WiP: ENH: More on submodules and distribution
================================================================================
Pull request is open, was reported by bpoldrack and has 85 comment(s).
<https://github.com/datalad/datalad/pull/388>

New PR due to the limitations on the number of commits.


Comments:
Traceback (most recent call last):
  File "/usr/bin/git-hub", line 2047, in <module>
    main()
  File "/usr/bin/git-hub", line 2041, in main
    args.run(parser, args)
  File "/usr/bin/git-hub", line 621, in check_config_and_run
    cmd.run(parser, args)
  File "/usr/bin/git-hub", line 1342, in run
    cls.print_issue(pull, c, ic)
  File "/usr/bin/git-hub", line 966, in print_issue
    for c in cls.merge_issue_comments(comments, review_comments):
  File "/usr/bin/git-hub", line 955, in merge_issue_comments
    comments.sort(key=lambda c: c['sort_key'])
  File "/usr/bin/git-hub", line 955, in <lambda>
    comments.sort(key=lambda c: c['sort_key'])
KeyError: 'sort_key'
````

on Debian sid with version 0.10-1

**Comment by leandro-lucarella-sociomantic** [comment]
Mmm, it seems like PR showing is completely broken due to some changes in GitHub API. Working on it.

**Comment by leandro-lucarella-sociomantic** [comment]
Could you test again with the fix in #177, it should be enough to download the new [git-hub](https://raw.githubusercontent.com/leandro-lucarella-sociomantic/git-hub/fix-pull-show/git-hub) script and use it like `./git-hub` or `python /path/to/git-hub`.

**Comment by hanke** [comment]
Works!

Thanks for the quick fix!
